COPYING

```diff
@@ -0,0 +1,502 @@
+                  GNU LESSER GENERAL PUBLIC LICENSE
```

Comment:
I don't believe this is the correct license file. I think this is inherited from the `master` branch of `flathub/flathub` and should be removed.